cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/store/SourceControlMetadataStore.java

```diff
@@ -64,6 +69,18 @@ private StructuredTable getNamespaceSourceControlMetadataTable() {
     return namespaceSourceControlMetadataTable;
   }
 
+  private StructuredTable getRepositorySourceControlMetadataTable() {
+    try {
+      if (repositorySourceControlMetadataTable == null) {
+        repositorySourceControlMetadataTable = context.getTable(
+            StoreDefinition.RepositorySourceControlMetadataStore.REPOSITORY_SOURCE_CONTROL_METADATA);
+      }
+    } catch (TableNotFoundException e) {
```

Comment:
throw `TableNotFoundException` directly as it is already `RuntimeException`

Reply:
Fixed. Thanks